atrium-scala/gradlew.bat

```diff
@@ -0,0 +1,89 @@
+@rem
```

Comment:
I'm sorry, this just changed in master (atrium-scala was renamed to atrium-scala2).
Please rebase your changes on master and re-do the update in atrium-scala2

btw. nice that you have spotted this, forgot to include it in the description :+1: